File: 백준/Bronze/2581. 소수/소수.java
import java.util.ArrayList;
import java.util.Scanner;

public class Main {
    public static void main(String[] args) {
        Scanner sc = new Scanner(System.in);
        ArrayList<Integer> factorList = new ArrayList<>();

        int n1 = sc.nextInt();
        int n2 = sc.nextInt();

        for (int i = n1; i <= n2; i++) {
            int cnt = 0;
            for (int j = 1; j <= i; j++) {
                if (i % j == 0) {
                    cnt++;
                }
            }

            if (cnt == 2) {
                factorList.add(i);
            }
        }

        if (factorList.isEmpty()) {
            System.out.println(-1);
        } else {
            int sum = factorList.stream().mapToInt(Integer::intValue).sum();
            int minNum = factorList.get(0);

            System.out.println(sum);
            System.out.println(minNum);
        }
    }
}

File: 백준/Bronze/2581. 소수/소수.cc
#include <iostream>
using namespace std;

int main() {

	int n2;
	int n1;
	int sum = 0;
	int minNum = 0;

	cin >> n1 >> n2;

	for (int i = n1; i <= n2; i++)
	{
		int cnt = 0;
		for (int j = 1; j <= i; j++) {
			if (i % j == 0)
			{
				cnt++;
			}
		}

		if (cnt == 2)
		{
			sum += i;

			if (minNum == 0)
			{
				minNum = i;
			}
		}
	}

	if (minNum == 0)
	{
		cout << -1 << endl;
	}
	else {
		cout << sum << "\n" << minNum << endl;
	}
}	